libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -264,16 +264,27 @@ public CredentialIssuerConfig getCriConfig(CriOAuthSessionItem criOAuthSessionIt
     public CredentialIssuerConfig getCriConfigForConnection(String connection, String criId) {
         final String pathTemplate =
                 ConfigurationVariable.CREDENTIAL_ISSUERS.getPath() + "/%s/connections/%s";
-        Map<String, String> result = getSsmParameters(pathTemplate, false, criId, connection);
-
-        if (result == null || result.isEmpty()) {
+        String parameter = getSsmParameter(resolvePath(pathTemplate, criId, connection));
+        if (parameter != null) {
+            try {
+                return objectMapper.readValue(parameter, CredentialIssuerConfig.class);
+            } catch (JsonProcessingException e) {
+                LOGGER.error(
+                        "Failed to parse credential issuer configuration at parameter path {} because: {}",
+                        pathTemplate,
+                        e);
+                throw new RuntimeException(e);
```

Comment:
I suspect Sonar will complain that this isn't a specific exception class - is there a convenient one we can use?

Reply:
Created a new `ConfigParseException` as there isn't one directly related to this that extends `RuntimeException`